### ISSUE TITLE
?

### ISSUE THREAD

**Issue by matheushdev** [KeyPharm/Anticansancio-NiNoKuniXW#1]
Would I have to use the game in full screen?

**Comment by KeyPharm** [comment]
Yes, at 1920x1080. I guess I could make it windowed, but different resolutions have different aspect ratios at window too. Its based on capturing the window and search for icons on it, so if you minimize the game it stops cliking it.

**Comment by matheushdev** [comment]
> Yes, at 1920x1080. I guess I could make it windowed, but different resolutions have different aspect ratios at window too. Its based on capturing the window and search for icons on it, so if you minimize the game it stops cliking it.

Here "arrowFound = hayFlechaEnPantalla()" is returning null.
Is it because the "img/volverAJugar.png" is missing ?

**Comment by matheushdev** [comment]
Do you have Discord?

**Comment by KeyPharm** [comment]
I did a small channel you can join
https://discord.gg/Cge9D6AF
